koa_crud/src/models/member.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-shadow */
+import { Schema, model, Document } from 'mongoose';
+
+export type MemberDocument = Document & {
+  username: string;
+  password: string;
+  realName: string;
```

Comment:
since realName is optional and could possibly have `null` value then please change this to:
```ts
realName?: string | null
```